packages/browser/src/plugins/page-enrichment/index.ts

```diff
@@ -214,6 +215,14 @@ class PageEnrichmentPlugin implements Plugin {
 
     evtCtx.userAgent = navigator.userAgent
 
+    try {
+      evtCtx.userAgentData = await clientHints(
+        this.instance.options.highEntropyValuesClientHints
```

Comment:
Similar to the original PR, can we cache the client hints during the plugin load? These fields won't change, so no reason to perform a potentially expensive operation on every event :smile: